drivers/volume/kdmp/kdmp.go

```diff
@@ -371,6 +375,24 @@ func (k *kdmp) GetBackupStatus(backup *storkapi.ApplicationBackup) ([]*storkapi.
 	}
 	return volumeInfos, nil
 }
+
+// Check whether to run kopia executor in debug mode
+func enableKopiaExecutorDebugMode() bool {
+	logrus.Info("enableKopiaExecutorDebugMode entering")
+	kdmpData, err := core.Instance().GetConfigMap(stork_driver.KdmpConfigmapName, stork_driver.KdmpConfigmapNamespace)
+	if err != nil {
+		logrus.Info("enableKopiaExecutorDebugMode unable to read the cm")
+		logrus.Tracef("error readig kdmp config map: %v", err)
```

Comment:
nit: Typo on readig

Reply:
Going to change it to, logrus.warnf and thinking of adding cm name as well, would that be better? @vsundarraj-px